_shared/project/bin/make_devdata

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+import subprocess
+import tempfile
+from pathlib import Path
+
+if __name__ == "__main__":
+    with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
I'm not aware of a naming convention of ours that this breaks? I think we just follow PEP 8 (so for variables: <q>lowercase, with words separated by underscores <b>as necessary to improve readability</b></q>) and look to the standard library and good third-party libraries for examples of how this guideline is applied.

So for example it's common to run something like `tmpdirname` together to keep it nice and short because underscores (`tmp_dir_name`) are judged not necessary to improve readability in this case (I think they actually harm readability in many cases).

When the examples in the docs for a library use a conventional name like `tmpdirname` I'll tend to reuse that unless it violates PEP 8 (if the docs had called the local variable `TmpDirName` I would not copy that).